packages/devtools_app/lib/src/screens/performance/panes/rebuild_stats/rebuild_stats_model.dart

```diff
@@ -276,17 +276,17 @@ class RebuildCountModel {
 }
 
 class Location {
-  Location({required this.id, this.path, this.line, this.column, this.name});
+  Location({required this.id, this.fileUri, this.line, this.column, this.name});
 
   final int id;
 
   /// Either all of path, line, column, and name are null or none are.
-  String? path;
+  String? fileUri;
```

Comment:
sorry for bad initial suggestion. Can we change this to fileUriString? This is consistent with other parts of DevTools where we follow this pattern:

- variables named "path" => /actually/a/file/path
- variables named "fileUriString" => "file:///this/is/a/string/that/equals/uri/toFilePath"
- variables named "uri" => this is a Uri object

Reply:
Ah yeah, that makes sense.